ote_sdk/ote_sdk/usecases/exportable_code/demo/requirements.txt

```diff
@@ -0,0 +1,3 @@
+openvino==2021.4
+openmodelzoo-modelapi @ git+https://github.com/openvinotoolkit/open_model_zoo/@master#egg=openmodelzoo-modelapi&subdirectory=demos/common/python
+ote-sdk @ git+https://github.com/openvinotoolkit/training_extensions/@develop#egg=ote-sdk&subdirectory=ote_sdk
```

Comment:
At installation time I see this error (it's actually a warning, as it proceeds successfully):
```
ERROR: ote-sdk 1.0 has requirement opencv-python==4.5.3.*, but you'll have opencv-python 4.5.4.60 which is incompatible.
```

And if wheel is not installed I see these messages as well:
```
  ERROR: Command errored out with exit status 1:
   command: venv/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-48fcardx/openmodelzoo-modelapi/demos/common/python/setup.py'"'"'; __file__='"'"'/tmp/pip-install-48fcardx/openmodelzoo-modelapi/demos/common/python/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-mfeshdkt
       cwd: /tmp/pip-install-48fcardx/openmodelzoo-modelapi/demos/common/python
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for openmodelzoo-modelapi
```

Reply:
What about opencv I suggest to upgrade version to 4.5.* for sdk.  If wheel is not installed, you use `python3 setup.py bdist_wheel` or another command? This working for me and I can add to README how to install without wheel package.

Reply:
>  I suggest to upgrade version to 4.5.* for sdk

It was causing lots of mypy issues, if I'm not mistaken. So, we won't do that now.

> If wheel is not installed, you use python3 setup.py bdist_wheel or another command?

Just `pip install demo_package-0.0-py3-none-any.whl` as in the readme. I guess simply adding `pip install wheel` do the readme should do the trick.